Source/WebCore/platform/graphics/ImageBuffer.cpp

```diff
@@ -80,6 +84,18 @@ RefPtr<ImageBuffer> ImageBuffer::create(const FloatSize& size, RenderingPurpose
     }
 #endif
 
+#if USE(SKIA)
+    static const char* enableCPURendering = getenv("WEBKIT_SKIA_ENABLE_CPU_RENDERING");
+    if (!enableCPURendering || !strcmp(enableCPURendering, "0")) {
+        static const char* disableAccelerated2DCanvas = getenv("WEBKIT_DISABLE_ACCELERATED_2D_CANVAS");
```

Comment:
It would be nicer to have these as runtime feature flags, but we can handle that in a separate patch... Or “patches” in plural, because at some point it would be nice to reduce the number of environment variables in favor of feature flags 😉